CONTRIBUTION.md

```diff
@@ -53,42 +41,102 @@ The format can be described more formally as follows:
+
+For example, a docs PR would look like
+
+```WINC-959: [docs] reorganizes readme```
+
+In doing this, you’ll be linking your PR to the Jira ticket for tracking.
+
+In the case of bug fix PRs, this will also automatically transition the associated Jira bug.
+
+- Opening the PR: `ASSIGNED` --> `POST`
+- PR merges: `POST` --> `MODIFIED`
+- `MODIFIED` to the `ON_QA` transition will have to be done manually by the PR author.
+- PR closes without merging, the Jira bug transitions back to NEW status.
+
+If you have no Jira issue or are not a Red Hat employee, your title should simply follow the format ```[<subsystem>] <title>```, with ```Fixes #<issue>``` in the comment. 
+
+### Open as draft
+
+The WMCO team opens pull requests as drafts before they are reviewed in order to prevent the tests from running right away
+[Open your PR as a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) before requesting reviews.
+
+
+## Merging a pull request
+
+Once your PR is open, you'll need at least two reviewers te get it merged.
+
+### Moving your PR out of draft
+
+Before moving your PR out of draft, you must have at least one /lgtm and one /approve on the PR.
+Once you have both, you are free to move it out of draft status by clicking the "ready for review" button
+in your PR interface. This will let your tests start running, and if they pass, your PR will merge automatically.
+
+### Respond to failing tests
+
+If your PR is experiencing a test failure that you believe to be a flake, feel free to run a retest.
+When retesting, be sure to add a link to the failing test, and a snippet of what caused the failure.
+The format should be
+
+```
+/retest-required
+
+<explanation of the error>
+<reason for retest>
+<prow.ci.openshift.org link> 
+
+```<log snippet of the failure> ```
+```
+
+Once your tests are passing, and your PR has an approval and a LGTM label, the CI should merge it automatically.
 
 ![Sample PR life-cycle](/images/PR-workflow.png)
+
+## Backporting changes
+Things like bugfixes should be backported to all [supported versions](https://access.redhat.com/support/policy/updates/openshift#windows).
+Once your bugfix is merged, go to the merged PR and comment 
+```/cherrypick <release>``` 
+to trigger the openshift-cherrypick-robot and create an automated backport.
+If you have multiple versions you need to backport to, go to each generated backport and run the 
+```/cherrypick <release>``` on that instead of master. This ensures the Jira work stays tracked properly. 
+
+If for example your PR to master needed to be backported to 4.11, 4.10, and 4.9, you would create an automatic
+backport against 4.11, go to that generated backport, and from there run ```/cherrypick 4.10```. Then, from 
+your 4.10 backport you would generate a backport for 4.9. 
+
+If the cherry pick bot fails, you will have to make your cherry picks manually.
```

Comment:
And execute the `/jira cherrypick OCBUGS-nnnn` command in manual PR to create the Jira associations..